samples/frameworks/electron/ts/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "electron-test-app",
+  "version": "1.0.0",
+  "description": "A sample Electron application",
+  "main": "App.ts",
```

Comment:
Is this the right main?

Reply:
Yup, that's the entry point for this app. Main is sort of overloaded in Electron (from my very limited understanding) since it can mean the `main` process (as opposed to `renderer` process) so I didn't know how to make that distinction better. 

I can have `main.ts` be the main entry point and call the main process class something else - but I am not sure what name would make sense

Reply:
Actually, I take that back - I removed app.ts and made main.ts the main entry point. It looks much better - will push that up shortly. Thanks Chris!